crates/bevy_ecs/src/world/mod.rs

```diff
@@ -788,6 +788,12 @@ impl World {
         unsafe { self.get_resource_with_id(component_id) }
     }
 
+    pub fn get_many_res<'w, MR: ManyRes<'w>>(&'w self) -> MR::Item {
```

Comment:
These definitely need doc examples: they're very useful, but it's hard to figure out how to use them from the type signatures.

Reply:
```suggestion
    pub fn get_many_resource<'w, MR: ManyRes<'w>>(&'w self) -> MR::Item {
```
the single version is called `get_resource`, it would be better to use the full word here too